python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -172,7 +197,15 @@ def build_airbyte_assets(
     tables = chain.from_iterable(
         chain([destination_tables], normalization_tables.values() if normalization_tables else [])
     )
-    outputs = {table: AssetOut(key=AssetKey(asset_key_prefix + [table])) for table in tables}
+    outputs = {
+        table: AssetOut(
+            key=AssetKey(asset_key_prefix + [table]),
+            metadata={"schema": MetadataValue.table_schema(schema_by_table_name[table])}
```

Comment:
I just remembered that some of our IOManagers explicitly look for a "schema" field in the output metadata to determine what database schema to store to / load from. So to avoid some weird potential bugs, maybe call this field "table_schema" or something

Reply:
do we want to do this for the [materializations too](https://github.com/dagster-io/dagster/blob/47254fc3623b05a8271574ec013bf5cd9ef762fd/python_modules/libraries/dagster-airbyte/dagster_airbyte/utils.py#L31)? they (at present) use `schema`